grpc/rust/BUILD

```diff
@@ -45,11 +45,10 @@ rust_library(
     name = "typedb_protocol",
     srcs = [
         ":typedb_protocol_src",
-        "lib.rs",
```

Comment:
This file is not necessary for the library compilation, and mixing generated and non-generated sources breaks rust library compilation as an external dependency.